dist/.goreleaser.yaml

```diff
@@ -205,7 +216,6 @@ dockers:
       - agent
     image_templates:
       - "scylladb/scylla-manager-agent:{{ .Version }}"
-      - "scylladb/scylla-manager-agent:latest"
```

Comment:
Why is this removed?

Reply:
Latest tag is related only to latest version which we deal during promotion . it's not needed here